contributing-docs/03_contributors_quick_start.rst

```diff
@@ -466,6 +466,11 @@ on macOS, install via
 
 2. Installing required Python packages
 
+.. note::
+  If pyenv and pyenv-virtualenv are not installed, install them first.
```

Comment:
Pyenv is just one of the ways you can install and activate venv. However I think we should likely change those instructions to use `pipx`. Pre-commit is better to be Installed via `pipx` as it will create and manage it's venv automatically. Would you like to change your PR and test it locally with pic instead?

Reply:
Thank you!
I changed the command to use pipx and I tested that I can configure pre-commit locally
<img width="497" alt="스크린샷 2024-06-18 오후 10 01 38" src="https://github.com/apache/airflow/assets/48010427/11dae188-196b-412d-8228-3e2371c6dab7">
<img width="580" alt="스크린샷 2024-06-18 오후 10 01 53" src="https://github.com/apache/airflow/assets/48010427/5da7d7f2-71cd-4001-8f3f-48c7ab1406ad">